lib/iex/lib/iex/pry.ex

```diff
@@ -116,6 +116,7 @@ defmodule IEx.Pry do
 
   The actual line is especially formatted in bold.
   """
+  @spec whereami(Path.t(), non_neg_integer(), integer()) :: {:ok, charlist()} | :error
```

Comment:
```suggestion
  @spec whereami(Path.t(), non_neg_integer(), pos_integer()) :: {:ok, IO.chardata()} | :error
```

This is actually more nested than just a charlist, [format_fragment/2](https://hexdocs.pm/elixir/IO.ANSI.html#format_fragment/2) returns chardata.